spec/services/types_generator_spec.rb

```diff
@@ -160,4 +162,80 @@
       end
     end
   end
+
+  describe 'cocina mapping' do
+    non_other_work_types = WorkType.all.reject { |work_type| work_type.id == 'other' }
```

Comment:
Using a local variable is a little gross here, but I can't use a `let()` variable given how I am using this outside test scope below.